src/domain/session/room/InviteViewModel.ts

```diff
@@ -105,44 +122,48 @@ export class InviteViewModel extends ViewModel {
         }
     }
 
-    _onInviteChange() {
-        this.emitChange();
+    _onInviteChange(): void {
+        this.emitChange("TODO");
```

Comment:
@MidhunSureshR need some guidance here for what string this should emit

Reply:
```suggestion
        this.emitChange("invite");
```

Reply:
Thanks. 

Can you explain to me in general, what are these for? Why do we do an `emitChange` that nothing is listening for?

Reply:
All views listen to `change` events on their viewmodel. In this case, the invite viewmodel will call `emitChange` when `invite` has some change and this causes the UI to do something (only if it wants to).

Reply:
> All views listen to change events on their viewmodel.

Hmm I see, can you point me to where that happens? Based on how you've stated it, I'd expect `InviteView` to be listening for `"change"` events coming from `InviteViewModel`, and updating its state accordingly, however I'm struggling to track down where in the code that takes place.

Reply:
See https://github.com/vector-im/hydrogen-web/blob/master/src/platform/web/ui/general/BaseUpdateView.ts#L61

Reply:
Ah I see it now. `TemplateView` inherits that, and most other views inherit `TemplateView`. Thank you!